IdentityCore/src/throttling/MSIDThrottlingService.m

```diff
@@ -112,7 +112,20 @@ + (BOOL)updateLastRefreshTimeDatasource:(id<MSIDExtendedTokenCacheDataSource>)da
                                  context:(id<MSIDRequestContext>)context
                                    error:(NSError **)error
 {
+    if([MSIDThrottlingService isThrottlingDisable])
+    {
+        return YES;
+    }
     return [MSIDThrottlingMetaDataCache updateLastRefreshTimeWithDatasource:datasource context:context error:error];
 }
 
++ (BOOL)isThrottlingDisable
```

Comment:
NIT: would if be ```isThrottlingEnable``` instead of ```isThrottlingDisable``` as I notice there are couple ```![MSIDThrottlingService isThrottlingDisable]``` turns out to be not disabled which is enabled

Reply:
+1

Reply:
After change it to `isThrottlingEnable ` I realized the number of places I need to add the not operator (!) are the same as the number of places able to remove that operator :) But anyway I updated the method name.